### DISCUSSION TITLE
Welcome to Cloudreve Discussions!

### DISCUSSION THREAD

**Discussion by HFO4** [cloudreve/cloudreve#1316] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Please use English in this seciton!**

**Comment by vvisionnn** [comment]
🎉🎉🎉

**Comment by elarbor** [comment]
这可真是个宝藏项目👍

**Comment by hhjeyoo** [comment]
社区版升级为PRO版需要重新部署还是在社区版的基础上输入授权码就行
